tests/bootstrap.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Bootstraps the Beans Tests.
+ *
+ * @package     Beans\BeansFramework\Tests
+ * @since       1.5.0
+ * @link        http://www.getbeans.io
+ * @license     GNU-2.0+
+ */
+
+if ( version_compare( phpversion(), '5.3.0', '<' ) ) {
+	die( 'Beans Unit Tests require PHP 5.3 or higher.' );
+}
+
+define( 'BEANS_TESTS_DIR', dirname( __FILE__ ) );
+
+define( 'BEANS_TESTS_LIB_DIR', BEANS_TESTS_DIR . '/../lib/' );
+
+/**
+ * You can turn on or off the integration tests.  Integration tests cause WordPress to load and run.
+ * PHP UnitTests, on the other hand, do not load WordPress, using Brain Monkey instead.
+ */
+define( 'BEANS_RUN_INTEGRATION_TESTS', false );
```

Comment:
Tests in this package are currently Unit Tests and not integration tests.  Therefore, we do not load WordPress.  Instead, we're using Brain/Monkey and mocking out WordPress functionality.

Reply:
I would advise not to run unit tests and integration tests together. We could use the phpunit `--testsuite` (or else) selection option to run integration tests separately.

Reply:
Example of a [possible directory setup](https://github.com/GaryJones/plugin-boilerplate/tree/master/tests), including the `bootstrap.php` which conditionally loads WP depending on the `--testsuite` that is run.